packages/RatPack.package/RPTestApp.class/instance/initializeSettings.st

```diff
@@ -0,0 +1,17 @@
+initialize-release
+initializeSettings
+
+	(self settings)
+		contentDecoders: (Dictionary newFrom: {
+			'test/contenttype' -> RPMockDecoder
+		});
+		
+		globalMiddleware: {
+			#RPTestMiddleware .
+			#RPGlobalHeaderMiddleware .
+			#RPContentParserMiddleware
+		} asOrderedCollection;
+		
+		routeNamingStrategies:  {
+			RPTestRouteNamingStrategy new
+		} asOrderedCollection.
```

Comment:
Why is this not failing if the only decoder we put is the MockDecoder ? Shouldn't then (because the ParserMiddleware is global all json post and put raise an error, because "missing decoder".
(I'm in a rush and try to find out later on by my self)

please check if this is smalltalk style and not c or c++

Reply:
All other tests have "requireCorrectContentParsing" set to false, so they don't care that we couldn't parse the content.

Reply:
This is the only thing I still disagree a bit, but thats not bad enough to block the merge.
I understand that this is more readable to you and probably to many others, but I think we should follow the styles of the language, even if only for the professorship.